src/primitive-field/primitive-field.component.ts

```diff
@@ -105,7 +105,7 @@ export class PrimitiveFieldComponent extends AbstractFieldComponent implements O
   }
 
   get tabIndex(): number {
-    return this.tabIndexService.getElemTabIndex(this.pathString);
+    return this.schema['disabled'] === true ? -1 : this.tabIndexService.getElemTabIndex(this.pathString);
```

Comment:
`disabled` could be either `true`or `false`, so for my comment, when it's true, it'll return `-1` and when it's `false` or `undefined` it'll return real `tabindex`.